Bioperl genbank2gff3

erasche opened the pull request:
Not all of the bioperl tools will be worth wrapping, especially when covered by other suites (e.g. seqtk), but some are definitely of interest. 

Done:

- genbank2gff3

Maybe todo? (Feel free to edit this list)

- bp_mutate
- bp_flanks
- ?

bgruening commented:
@erasche do you want to support this over time?
I'm more inclined to bet on biopython and work further on this: https://github.com/bgruening/galaxytools/tree/master/tools/biopython/converters

erasche commented:
@bgruening there's simply no biopython alternatives to this stuff. :/ So,
yes. I'm willing to support it because I desperately need it for chado.

Updates to your converters coming soon btw :-)

søn. 16. aug. 2015, 14.36 skrev Björn Grüning <notifications@github.com>:

> @erasche <https://github.com/erasche> do you want to support this over
> time?
> I'm more inclined to bet on biopython and work further on this:
> https://github.com/bgruening/galaxytools/tree/master/tools/biopython/converters
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/galaxyproject/tools-iuc/pull/247#issuecomment-131552688>
> .
>

erasche commented:
@bgruening given your comments, I've:

- moved out of WIP
- removed anything else from the TODO list. I'll leave the rest of the bioperl suite alone, but I definitely need this tool.

:)

bgruening commented on tools/bioperl/bp_genbank2gff3.xml:
```diff
@@ -0,0 +1,72 @@
+<tool id="bp_@EXECUTABLE@" name="BioPerl @EXECUTABLE@" version="1.0">
```
I would prefer a more readable name. I don't think that our users care about the underlying software (BioPerl).
`GenBank to gff3 converter`?

bgruening commented on tools/bioperl/bp_genbank2gff3.xml:
```diff
@@ -0,0 +1,72 @@
+--outdir -
+--ethresh $ethresh
+$model
+--typesource $typesource
```
"${typesource}"

bgruening commented on tools/bioperl/bp_genbank2gff3.xml:
```diff
@@ -0,0 +1,72 @@
+**What it does**:
+
+Convert genbank to gff3
+      ]]></help>
```
BioPerl ref?

bgruening commented on tools/bioperl/bp_genbank2gff3.xml:
```diff
@@ -0,0 +1,72 @@
+    <conditional name="sofile" label="Sequence Ontology">
+        <param name="sofile" label="Sequence Ontology File" type="select">
+            <option value="__none__" selected="True">None specified</option>
+            <option value="live">Latest SO</option>
```
s/SO/Sequence Ontolog/

erasche commented:
Ping @bgruening

bgruening commented:
And again a PR I missed. Sorry Eric!

erasche commented:
no problem :) thanks!
